animation.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <style>
+      * {
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+      }
+
+      .container {
+        width: 100%;
+        height: 80vh;
+        background-image: url("d-wallpaper-bg.jpg");
+        position: relative;
+        overflow-x: hidden;
+      }
+
+      .fog {
+        height: 300px;
+        width: 500%;
+        display: block;
+        background-image: url("fog.png");
+        top: 0;
+        left: 0;
+        right: 0;
+        background-repeat: repeat-x;
+        animation: fog 8s linear infinite;
+      }
+
+      @keyframes fog {
+        100% {
+          transform: translateX(-300px);
+        }
+      }
+
+      .blue-car {
+        left: 75%;
+        position: absolute;
+        bottom: 210px;
+      }
+
+      .blue-car img {
+        width: 100%;
+        height: 150px;
+        animation: bluecar 5s linear infinite;
+        -webkit-animation-iteration-count: 1;
+      }
+
+      @keyframes bluecar {
+        100% {
+          transform: translatex(-1500px);
+        }
+      }
+
+      .red-car {
+        left: 50px;
+        position: absolute;
+        bottom: 190px;
+        z-index: 3;
+      }
+
+      .red-car img {
+        width: 100%;
+        height: 150px;
+        animation: car 5s linear infinite;
+        -webkit-animation-iteration-count: 1;
+      }
+
+      @keyframes car {
+        100% {
+          transform: translatex(1500px);
+        }
+      }
+
+      .wheel {
+        left: 50px;
+        position: absolute;
+        bottom: 200px;
+        z-index: 3;
+      }
+
+      .wheel img {
+        width: 40px;
+        height: 40px;
+        animation: wheel 5s linear infinite;
+        -webkit-animation-iteration-count: 1;
+      }
+
+      .back-wheel {
+        left: 40px;
+        bottom: 5px;
+        position: absolute;
+      }
+
+      .front-wheel {
+        left: 238px;
+        bottom: 5px;
+        position: absolute;
+      }
+
+      @keyframes wheel {
+        100% {
+          transform: translatex(1500px) rotate(360deg);
+        }
+      }
+
+      .right-car-wheel {
+        left: 75%;
+        position: absolute;
+        bottom: 200px;
+        z-index: 2;
+      }
+
+      .right-car-wheel img {
+        width: 40px;
+        height: 40px;
+        animation: bluewheel 5s linear infinite;
+        -webkit-animation-iteration-count: 1;
+      }
+
+      .back-wheel-blue {
+        left: 39px;
+        bottom: 20px;
+        position: absolute;
+      }
+
+      .front-wheel-blue {
+        left: 220px;
+        bottom: 20px;
+        position: absolute;
+      }
+
+      @keyframes bluewheel {
```

Comment:
Check if we can use the same keyframes for all the wheels since the direction is different only.

Reply:
I can't find that approach it would be great if you share!